manifests/init.pp

```diff
@@ -343,4 +343,10 @@
 
   contain 'foreman::settings' # lint:ignore:relative_classname_inclusion (PUP-1597)
   Class['foreman::database'] -> Class['foreman::settings']
+
+  file { '/usr/share/foreman/tmp/updated_plugins':
```

Comment:
for Debian, this file is created via https://github.com/theforeman/foreman-packaging/pull/7853
for RPM, this needs implementatin